certbot/main.py

```diff
@@ -306,16 +293,24 @@ def _find_domains_or_certname(config, installer):
     return domains, certname
 
 
-def _report_new_cert(config, cert_path, fullchain_path):
+def _report_new_cert(config, lineage=None, print_fullchain_path=True):
     """Reports the creation of a new certificate to the user.
```

Comment:
Also, pretty sure we don't actually need `print_fullchain_path` or the conditional below, as per https://github.com/certbot/certbot/issues/4182#issuecomment-278158360.